### ISSUE TITLE
RecipeIngredient Class

### ISSUE THREAD

**Issue by Gufraan786** [DMU-CTEC2902/TeamTwistedFlax#15]
Hi, I was thinking in the RecipeIngredient Class we only include Recipe and Ingredient, so I was thinking if I could change RecipeContext because it includes all the parts?

**Comment by Gufraan786** [comment]
@DaveGerrardDMU @NjoadAlghanim @AaronMulgrew @MariamBajahzar In the Tools class I don't think we need Description because the users already know what items are? What do you think?

**Comment by AaronMulgrew** [comment]
But where does it say the description, like what tool it is?



On Mon, May 2, 2016 at 8:17 AM -0700, "Gufraan Arkate" <notifications@github.com<mailto:notifications@github.com>> wrote:


@DaveGerrardDMU<https://github.com/DaveGerrardDMU> @NjoadAlghanim<https://github.com/NjoadAlghanim> @AaronMulgrew<https://github.com/AaronMulgrew> @MariamBajahzar<https://github.com/MariamBajahzar> In the Tools class I don't think we need Description because the users already know what items are? What do you think?

-

**Comment by Gufraan786** [comment]
In the Tool class it says ToolId, Name, Description and Comments but,I think we only need name of Tool?

**Comment by AaronMulgrew** [comment]
Yep go for it, I don't see any need for a detailed description of a whisker anyway.



On Mon, May 2, 2016 at 8:34 AM -0700, "Gufraan Arkate" <notifications@github.com<mailto:notifications@github.com>> wrote:


In the Tool class it says ToolId, Name, Description and Comments but,I think we only need name of Tool?

-

**Comment by AaronMulgrew** [comment]
Is it committed now? The class with the recipeingredients in?

**Comment by Gufraan786** [comment]
No not yet, I am getting an error when logging in so will have to first fix that!

**Comment by AaronMulgrew** [comment]
Ok, an error related to the recipes?



On Mon, May 2, 2016 at 8:51 AM -0700, "Gufraan Arkate" <notifications@github.com<mailto:notifications@github.com>> wrote:


No not yet, I am getting an error when logging in so will have to first fix that!

-

**Comment by Gufraan786** [comment]
@MariamBajahzar Mariam is having a look at it and assisting me :)
